ruby/example_code/dynamodb/dynamodb_ruby_example_read_multiple_items.rb

```diff
@@ -0,0 +1,161 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This file is licensed under the Apache License, Version 2.0 (the "License").
+   You may not use this file except in compliance with the License. A copy of
+   the License is located at
+
+   http://aws.amazon.com/apache2.0/
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations under the License.
+*/
+
+# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+# snippet-sourceauthor:[Doug-AWS]
+# snippet-sourcedescription:[Gets an item from a DynamoDB table.]
+# snippet-keyword:[Amazon DynamoDB]
+# snippet-keyword:[get_item method]
+# snippet-keyword:[Ruby]
+# snippet-sourcesyntax:[ruby]
+# snippet-service:[dynamodb]
+# snippet-keyword:[Code Sample]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2018-03-16]
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require 'aws-sdk-dynamodb'  # v2: require 'aws-sdk'
+require 'rspec'
+# Create dynamodb client in us-west-2 region
+dynamodb = Aws::DynamoDB::Client.new(region: 'us-west-2')
+
+# Enables working with DynamoDB attribute values using hashes, arrays, sets, integers, floats, booleans, and nil.
+# Disabling this option requires that all attribute values have their types specified, e.g. `{ s: 'abc' }`
+# instead of simply `'abc'`.
+@options [Boolean] :simple_attributes (true)
```

Comment:
simple attributes defaults to true - be aware it modifies the response to be in a different format. For these examples I would make sure it is leveraged - https://github.com/aws/aws-sdk-ruby/blob/master/gems/aws-sdk-dynamodb/lib/aws-sdk-dynamodb/plugins/simple_attributes.rb